src/librustc_error_codes/error_codes/E0706.md

```diff
@@ -0,0 +1,78 @@
+ `async fn`s are not yet supported in Rust.
```

Comment:
When I read this I was like "but it is?" -- can we clarify that this is about traits early on?